.github/workflows/build-and-test.yml

```diff
@@ -86,6 +87,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        # Needed by sonar to get the git history for the branch the PR will be merged into.
+        with:
+          fetch-depth: 0
```

Comment:
Sonarqube was erroring out like [this](https://github.com/DataBiosphere/terra-billing-profile-manager/actions/runs/4621998118/jobs/8174213641#step:9:38) on PRs until I added this.